libraries/rib-base/src/main/java/com/badoo/ribs/android/dialog/DialogExtensions.kt

```diff
@@ -71,6 +77,21 @@ private fun AlertDialog.Builder.setTexts(dialog: Dialog<*>) {
     dialog.message?.let { setMessage(it.resolve(context)) }
 }
 
+private fun AlertDialog.markTitleAsHeading() {
+    findViewById<TextView>(android.R.id.title)?.let { titleView ->
```

Comment:
There is `public void setAccessibilityHeading(boolean isHeading)` method on any view, no need to use delegate.

Reply:
@CherryPerry this is only available on API 28. There might be an app company version of it though

Reply:
@LachlanMcKee just fixed it using `ViewCompat.setAccessibilityHeading` - even if it's no-op for <28 we're appending it for all API levels that support headings